src/pde.cpp

```diff
@@ -151,8 +172,64 @@ void PDE::set_solution (
     _solution_function->interpolate(*constant_fn);
   }
 
-  _bilinear_form->set_coefficient("uu", _solution_function);
-  _linear_form->set_coefficient("uu", _solution_function);
+  _bilinear_form->set_coefficient(_variable, _solution_function);
+  _linear_form->set_coefficient(_variable, _solution_function);
+}
+//--------------------------------------
+void PDE::set_solutions (
+  std::vector<Linear_Function> expression
```

Comment:
We need a way to have 
`void PDE::set_solutions (std::vector<RandomExpression> expression )`
If we have std::vector<dolfin::Expression> expression as an argument can we still call it with our own Expression class ?